opcua/client.py

```diff
@@ -307,3 +307,6 @@ def get_namespace_array(self):
     def get_namespace_index(self, uri):
         uries = self.get_namespace_array()
         return uries.index(uri)
+
+    def history_read(self, params):
+        return self.bclient.history_read(params)
```

Comment:
Then you can add a simple history_read(node, start_time, end_time) to Client if you think it is usefull